src/commands/convert/index.ts

```diff
@@ -17,20 +39,34 @@ function getInputFilename(): string {
 }
 
 export default class Convert extends Command {
-  static description = 'The generic convert command translates any supported file-based security results set into the Heimdall Data Format'
+  static description =
+    'The generic convert command translates any supported file-based security results set into the Heimdall Data Format';
 
-  static examples = ['saf convert -i input -o output']
+  static examples = ['saf convert -i input -o output'];
```

Comment:
this is one such example of there being editor settings in effect.  unfortunately, the linter for the saf cli is currently configured to disallow semicolons.  i find it annoying, but haven't gotten around to changing the configuration since it'd require fixing a whole lot of other files.